packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -39,6 +40,42 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     },
   });
 
+  //cancel reminders for cancelled/rescheduled bookings that are scheduled within the next hour
+  const remindersToCancel = await prisma.workflowReminder.findMany({
+    where: {
+      cancelled: true,
+      scheduledDate: {
+        lte: dayjs().add(1, "hour").toISOString(),
+      },
+    },
+  });
+
+  try {
+    const workflowRemindersToDelete: Prisma.Prisma__WorkflowReminderClient<WorkflowReminder, never>[] = [];
+
+    for (const reminder of remindersToCancel) {
+      await client.request({
```

Comment:
NIT, what happens when this call fails? Perhaps an idea to catch the request error and throw